testdata/scripts/debugdir.txt

```diff
@@ -0,0 +1,18 @@
+garble -debugdir ./test1 build
+dircmp 'test1' 'test1/test/imported/z0.go' 'test1/main/z0.go'
```

Comment:
you could just do `exists test1/test/imported/z0.go` and such, no need for a custom command.

I would also do simple sanity checks, like `! grep test1/test/imported/z0.go 'ImportedFunc'`.